mypaas/client/_push.py

```diff
@@ -57,7 +57,8 @@ def push(domain, dockerfile):
                     zf.write(filename, fname_in_zip)
         zf.write(dockerfile, "Dockerfile")  # the deploy will simply use "Dockerfile"
     payload = f.getvalue()
-
+    fileSize = f.getbuffer().nbytes
+    print(f":heavy_check_mark: Deploy file size is {fileSize / 1_000_000} MB.")
```

Comment:
I prefer MiB since we use that in a few other places in this lib. 
```suggestion
    print(f":heavy_check_mark: Deploy file size is {fileSize / 2**20} MiB.")
```